apps/meteor/client/views/admin/apps/AppsList.tsx

```diff
@@ -1,202 +1,26 @@
-					itemsPerPage={itemsPerPage}
-					onSetItemsPerPage={onSetItemsPerPage}
-					onSetCurrent={onSetCurrent}
-					paginationProps={paginationProps}
-					isMarketplace={isMarketplace}
-				/>
-			)}
-
-			{noMarketplaceOrInstalledAppMatches && (
-				<Box mbs='x20'>
-					<States>
-						<StatesIcon name='magnifier' />
-						<StatesTitle>{t('No_app_matches')}</StatesTitle>
-						{appsResult?.value?.shouldShowSearchText ? (
-							<StatesSubtitle>
-								{t('No_marketplace_matches_for')}: <strong>"{text}"</strong>
-							</StatesSubtitle>
-						) : (
-							''
-						)}
-						<StatesSuggestion>
-							<StatesSuggestionText>{t('You_can_try_to')}:</StatesSuggestionText>
-							<StatesSuggestionList>
-								<StatesSuggestionListItem>{t('Search_by_category')}</StatesSuggestionListItem>
-								<StatesSuggestionListItem>{t('Search_for_a_more_general_term')}</StatesSuggestionListItem>
-								<StatesSuggestionListItem>{t('Search_for_a_more_specific_term')}</StatesSuggestionListItem>
-								<StatesSuggestionListItem>{t('Check_if_the_spelling_is_correct')}</StatesSuggestionListItem>
-							</StatesSuggestionList>
-						</StatesSuggestion>
-					</States>
-				</Box>
-			)}
-
-			{noInstalledAppMatches && (
-				<Box mbs='x20'>
-					<States>
-						<StatesIcon name='magnifier' />
-						<StatesTitle>{t('No_installed_app_matches')}</StatesTitle>
-						{appsResult?.value?.shouldShowSearchText ? (
-							<StatesSubtitle>
-								<span>
-									{t('No_app_matches_for')} <strong>"{text}"</strong>
-								</span>
-							</StatesSubtitle>
-						) : (
-							''
-						)}
-						<StatesSuggestion>
-							<StatesSuggestionText>{t('Try_searching_in_the_marketplace_instead')}</StatesSuggestionText>
-						</StatesSuggestion>
-						<StatesActions>
-							<StatesAction onClick={(): void => marketplaceRoute.push({ context: '' })}>{t('Search_on_marketplace')}</StatesAction>
-						</StatesActions>
-					</States>
-				</Box>
-			)}
-
-			{noInstalledAppsFound && (
-				<Box mbs='x20'>
-					<States>
-						<StatesIcon name='magnifier' />
-						<StatesTitle>{t('No_apps_installed')}</StatesTitle>
-						<StatesSubtitle>{t('Explore_the_marketplace_to_find_awesome_apps')}</StatesSubtitle>
-						<StatesActions>
-							<StatesAction onClick={(): void => marketplaceRoute.push({ context: '' })}>{t('Explore_marketplace')}</StatesAction>
-						</StatesActions>
-					</States>
-				</Box>
-			)}
-
-			{appsResult.phase === AsyncStatePhase.REJECTED && (
-				<Box mbs='x20'>
-					<States>
-						<StatesIcon variation='danger' name='circle-exclamation' />
-						<StatesTitle>{t('Connection_error')}</StatesTitle>
-						<StatesSubtitle>{t('Marketplace_error')}</StatesSubtitle>
-						<StatesActions>
-							<StatesAction onClick={reload}>
-								<Icon mie='x4' size='x20' name='reload' />
-								{t('Reload_page')}
-							</StatesAction>
-						</StatesActions>
-					</States>
-				</Box>
-			)}
-		</>
-	);
-};
+} & ComponentProps<typeof Box>;
```

Comment:
`ComponentProps<typeof Box>` is a whole lot of props... do you really want to add all that?

Reply:
Solved